04_numeronym/NimishMudhale/main_test.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMainOutput(t *testing.T) {
+	// Given
+	r := require.New(t)
+	var buf bytes.Buffer
+	out = &buf
+	// When
+	main()
+	// Then
+	expected := fmt.Sprint([]string{"a11y", "K8s", "abc"})
+	actual := buf.String()
+	r.Equalf(expected, actual, "Unexpected output in main()")
+}
+
+var cases = []struct {
+	in, out []string
+}{
+	{[]string{"internationalization", "Administration", "consumability"}, []string{"i18n", "A12n", "c11y"}},
+	{[]string{}, []string{}},
+	{[]string{""}, []string{""}},
+	{[]string{"🦄☣🦄☣"}, []string{"ð12£"}},
```

Comment:
Does this test pass?
It looks like it should fail the test, I'm not seeing the ð, £ and 12 characters in between in  "🦄☣🦄☣".

Reply:
done